Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN if [ "${RELEASE_MODE}" = true ]; then \
     else gradle build --exclude-task test -Dorg.gradle.project.version=${APP_VERSION}; fi
 
 # For ARM build use flag: `--platform linux/arm64`
-FROM --platform=$BUILDPLATFORM amazoncorretto:21.0.2
```

Comment:
Why do we remove Buildplatform from Dockerfile?

Reply:
up

Reply:
Alredy fixed